src/util/reactTestingLibraryExtended.js

```diff
@@ -1,17 +1,28 @@
 import React from 'react'
 import { render } from '@testing-library/react'
 import { MemoryRouter } from 'react-router'
+import { Provider } from 'react-redux'
+import { createStore } from 'redux'
+import rootReducer from '../store/reducers'
+import createMiddleware from '../store/middleware'
+import '@testing-library/jest-dom/extend-expect' // Added many additional test helpers https://github.com/testing-library/jest-dom
 
-const AllTheProviders = ({ children }) => {
+export function generateStore (history, initialState) {
+  return createStore(rootReducer, initialState, createMiddleware(history))
```

Comment:
I feel like we should be initializing the store exactly as we do in the app: via `store/index`. Why not?

Reply:
Gotcha. I do still think there may be something to do here that is more directly employs the actual initial setup of the app that would be simplifying to complexifying (i.e. actually mocking the `fetchCurrentUser` graphql result in MSW)... Anyway, I'll save it and maybe before our Monday sync on this I'll make a little PR atop this work to try it out and--if it looks good--can be part of the discussion on Monday.